packages/key-utils/src/functions.ts

```diff
@@ -28,7 +28,7 @@ export const logger = Loggers.DEFAULT.get('sphereon:key-utils')
  * @param kms. Optional KMS to use. If provided will be the returned name. Otherwise the default KMS will be returned
  */
 export const getKms = async (context: IAgentContext<any>, kms?: string): Promise<string> => {
-  if (kms) {
+  if (kms && kms !== 'default') {
```

Comment:
Huh. If I named a KMS default, then it should be returned. The below code already handles the case where we can determine a default KMS. So if we use a KMS (which we won't soon anymore) that does not have the option to determine a default we should throw an error. 

So what is this currently fixing? Because if we still use a Veramo Key management system, we should be using the Sphereon Key Management System

Reply:
Okay this has been reverted it seems

Reply:
yes reverted a while ago when I learned about keyManagerGetDefaultKeyManagementSystem()